src/modules/settings.ts

```diff
@@ -5,12 +5,19 @@ export const enum LabelScheme {
   RiskLayer = 'RiskLayer'
 }
 
+export const enum Interpolation {
```

Comment:
I started with using a checkbox that only allows turning on or off linear interpolation but that didn't go very well with the rest of the settings dialog code.